rclcpp/include/rclcpp/logger.hpp

```diff
@@ -34,8 +35,7 @@
  * This should be used in combination with `RCLCPP_LOG_MIN_SEVERITY` to compile
  * out logging macros.
  */
-// TODO(dhood): determine this automatically from `RCLCPP_LOG_MIN_SEVERITY`
-#ifndef RCLCPP_LOGGING_ENABLED
+#ifdef RCLCPP_LOGG_MIN_SEVERITY_NONE
```

Comment:
typo  
RCLCPP_LOGG_MIN_SEVERITY_NONE -> RCLCPP_LOG_MIN_SEVERITY_NONE

Reply:
That sounds more correct, I'll change that up soon.

Reply:
Should we set LOGGING_ENABLED to 1 based on any other LOG_MIN_SEVERITY?